Formula/libbitcoin-explorer.rb

```diff
@@ -0,0 +1,80 @@
+class LibbitcoinExplorer < Formula
+  desc "Bitcoin Command-line Tool"
+  homepage "https://github.com/libbitcoin/libbitcoin-explorer"
+  url "https://github.com/libbitcoin/libbitcoin-explorer/archive/v3.3.0.tar.gz"
+  sha256 "029dc350497bdaad4d8559f7954405011b9e1b996aa4d4cc124f650e2eca00a6"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "zeromq"
+  depends_on "libbitcoin"
+
+  resource "secp256k1" do
+    url "https://github.com/libbitcoin/secp256k1/archive/v0.1.0.13.tar.gz"
```

Comment:
I want to say that the fork will be creating tags regularly. The libbitcoin team has a much better idea of which exact version to use. Is there a reason to prefer the upstream repo?